spec/system/candidate_interface/candidate_personal_details_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering their personal details' do
+  scenario 'Logged in candidate with no personal details' do
+    given_i_am_signed_in
+    and_i_visit_the_site
+    when_i_click_on_personal_details
+    then_i_can_edit_my_personal_details
+  end
+
+  scenario 'Candidate can fill in personal details' do
+    given_i_am_on_the_personal_details_page
+    when_i_fill_in_the_form
+    then_i_can_check_my_answers
+  end
+
+  scenario 'Candidate can go back to their application' do
+    given_i_am_on_the_personal_details_page
+    when_i_click_on_back_to_application
+    then_i_can_see_my_application
+  end
+
+  def given_i_am_signed_in
+    candidate = FactoryBot.create(:candidate)
+    login_as(candidate)
+  end
+
+  def given_i_am_on_the_personal_details_page
+    given_i_am_signed_in
+    visit candidate_interface_personal_details_edit_path
+  end
+
+  def and_i_visit_the_site
+    visit candidate_interface_application_form_path
+  end
+
+  def when_i_click_on_personal_details
+    click_link 'Personal details' # TODO: t('string')
+  end
+
+  def then_i_can_edit_my_personal_details
+    expect(page).to have_content 'Personal details' # TODO: t('string')
+    expect(page).to have_content 'First name' # TODO: t('string')
+    expect(page).to have_content 'Last name' # TODO: t('string')
```

Comment:
We should probably replace these TODOs in this file with translation strings

Reply:
Makes sense! Done. 🚀